fili-core/src/main/java/com/yahoo/bard/webservice/druid/model/filter/InFilter.java

```diff
@@ -0,0 +1,69 @@
+// Copyright 2016 Yahoo Inc.
+// Licensed under the terms of the Apache license. Please see LICENSE.md file distributed with this work for terms.
+package com.yahoo.bard.webservice.druid.model.filter;
+
+import com.yahoo.bard.webservice.data.dimension.Dimension;
+
+import java.util.Collections;
+import java.util.List;
+import java.util.Objects;
+
+import javax.validation.constraints.NotNull;
+
+/**
+ * Fili model of the Druid In Filter: http://druid.io/docs/0.9.1.1/querying/filters.html.
+ * <p>
+ * An In Filter is a generalization of the {@link SelectorFilter}. Rather than filtering on a specific value for a
+ * specific diimension, the In Filter takes a list of values. A dimension satisfies the In Filter iff its value is
+ * contained in the specified list.
```

Comment:
May be worth mentioning that an `in` filter is the same as an `or` filter around a collection of `selector` filters. (I think that's right, right?)